extract.go

```diff
@@ -188,11 +215,26 @@ func (e *entityExtracter) chunk(tokens []*Token) []Entity {
 	return entities
 }
 
-func (m *binaryMaxentClassifier) encode(features map[string]string, label string) []encodedValue {
-	encoding := []encodedValue{}
-	for _, key := range featureOrder {
-		val := features[key]
-		entry := strings.Join([]string{key, val, label}, "-")
+func (m *binaryMaxentClassifier) byteJoin(a, b, c string, rjc rune) string {
+	jc := byte(rjc)
```

Comment:
Can we just use `var jc byte = '-'`, and get rid of the `rjc` parameter?

Reply:
Yeah, I thought about using it elsewhere but it didn't give as many savings as I thought. So probably worth doing.